### ISSUE TITLE
Delete article via API results in request.ERROR: Uncaught PHP Exception

### ISSUE THREAD

**Issue by Strubbl** [wallabag/wallabag#3608]
### Issue details

Delete an article via the API, e.g. with the Android app. You see an error in the log, which is exposed via the API resulting in ugly error messages being returned to the user. But the article is deleted.

```
[2018-03-19 08:03:37] request.INFO: Matched route "api_delete_entries". {"route":"api_delete_entries","route_parameters":{"_controller":"Wallabag\ApiBundle\Controller\EntryRestController::deleteEntriesAction","_format":"json","entry":"964","_route":"api_delete_entries"},"request_uri":"https://wallabag.anbuco.fr/api/entries/964.json","method":"DELETE"} []
[2018-03-19 08:03:37] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "Wallabag\CoreBundle\Entity\Entry object not found." at /var/www/wallabagv2/vendor/sensio/framework-extra-bundle/Request/ParamConverter/DoctrineParamConverter.php line 66 {"exception":"[object] (Symfony\Component\HttpKernel\Exception\NotFoundHttpException(code: 0): Wallabag\CoreBundle\Entity\Entry object not found. at /var/www/wallabagv2/vendor/sensio/framework-extra-bundle/Request/ParamConverter/DoctrineParamConverter.php:66)"} []
```
"This error does not seem to affect the actual deletion of the article."

The issue was reported in https://github.com/wallabag/android-app/issues/646

### Environment

* wallabag version (or git revision) that exhibits the issue:2.3.2
* How did you install wallabag? Via `git clone` or by downloading the package?
* Last wallabag version that did not exhibit the issue (if applicable):
* php version:
* OS:Debian
* type of hosting (shared or dedicated):
* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL):

### Steps to reproduce/test case

Delete an article via the Android app.

**Comment by j0k3r** [comment]
Regarding the error it seems that the entry was already deleted.
Maybe the user has clicked twice?

**Comment by Strubbl** [comment]
No. I can reproduce the issue with the app while only clicking it only once. 

In the referenced issue, @Kdecherf already confirmed that the issue is still open and has not been cleared by his commits with PR #3516. Unfortunately.

**Comment by Kdecherf** [comment]
This exception is a normal exception when two `DELETE` requests are sent for the same entry.

It appears that when the android app throws the exception seen on wallabag/android-app#646, it fires another `DELETE` request which eventually leads to a 404 error on the server side (_confirmed with a tcpdump capture_).

I will open a new issue regarding the root cause of wallabag/android-app#646
